.github/workflows/cli-regression.yml

```diff
@@ -0,0 +1,40 @@
+name: CLI Regression Test
+
+on:
+  push:
+    branches:
+      - '**'
+
+# Avoid duplicate workflows on same branch
+concurrency: 
+  group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
nit: should this be given a `-cli-regression` suffix like other workflows?

Reply:
Suffix only needed for workflows that can be triggered by another workflow (have the `on: workflow_call` trigger event). The `github.workflow` name here will always be `cli-regression` and is okay without the additional suffix. 

Ex: When `cypress` & `python` workflows are called from the `nightly` workflow, each of their `github.workflow` values is `nightly`, so need suffix to distinguish them.